tools/rosbag/README.md

```diff
@@ -17,6 +17,8 @@ rosbag record --file-name record.yaml
 # topic_name: freq(double)
 # topic_name: -1 (it will be recorded at publish freq)
 #           : 0 (it will not be recorded)
+#           : -2 (for latched/[pseudo-latched](https://github.com/rapyuta-robotics/ros_comm/pull/10#issue-463244616) topics, to keep them in every bag split)
```

Comment:
Minor, but this link isn't actually displayed as [pseudo-latched](https://github.com/rapyuta-robotics/ros_comm/pull/10#issue-463244616) because this is inside a commented region.

Reply:
I know ... I just kept it like this because I think this is more readable.